x/liquidity/legacy/v4/store.go

```diff
@@ -0,0 +1,21 @@
+package v4
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func DeleteMMOrderIndexes(store sdk.KVStore) {
+	iter := sdk.KVStorePrefixIterator(store, MMOrderIndexKeyPrefix)
+	defer iter.Close()
+
+	for ; iter.Valid(); iter.Next() {
+		store.Delete(iter.Key())
+	}
+}
+
+func MigrateStore(ctx sdk.Context, storeKey sdk.StoreKey, cdc codec.BinaryCodec) error {
```

Comment:
I think we need to add `paramSpace.Set` for the new param's key  [Ref.](https://github.com/crescent-network/crescent/blob/0684439eb6cedde0f62247852fe0251b1d1e97c2/x/mint/legacy/v2/store.go#L10-L17)

Reply:
OK, I was setting the new parameter in the upgrade handler. I'll move it to `MigrateStore`!